src/frequenz/sdk/actor/power_distributing/_battery_pool_status.py

```diff
@@ -72,13 +72,16 @@ class BatteryPoolStatus:
     def __init__(
         self,
         battery_ids: Set[int],
+        battery_status_sender: Sender[BatteryStatus],
         max_data_age_sec: float,
         max_blocking_duration_sec: float,
     ) -> None:
         """Create BatteryPoolStatus instance.
 
         Args:
             battery_ids: set of batteries ids that should be stored in pool.
+            battery_status_sender: Channel for sending information which batteries are
+                working.
```

Comment:
This is not really a channel... Maybe:

```suggestion
            battery_status_sender: The sender used for sending the status of the
                batteries in the pool.
```

Reply:
Done